scripts/release/release-test.R

```diff
@@ -35,7 +36,12 @@ library(stringi)
 library(MolgenisArmadillo)
 library(DSI)
 library(dsBaseClient)
+
+# FIXME: waiting for PR https://github.com/molgenis/molgenis-r-datashield/pull/62
+#        needed for https://github.com/molgenis/molgenis-service-armadillo/pull/277
+#devtools::install("/Users/clemens/Documents/GitHub/molgenis-r-datashield/")
```

Comment:
Is this still required?

Reply:
Latest https://github.com/molgenis/molgenis-r-datashield/tags has https://github.com/molgenis/molgenis-r-datashield/pull/62 in it. Not sure how long it takes for CRAN to publish.

https://github.com/molgenis/molgenis-service-armadillo/pull/277 is fixed and we are not yet releasing.

We can let it in and cleanup for the next step. OK?